pyextract.py

```diff
@@ -250,6 +259,18 @@ def __gunzip_all__(self):
         print("\n" + Highlight.Convert("gunzip") + " all finish")
         return 0
 
+    def extract_coredump(self):
+        coredump = self.__find_coredump()
+        if len(coredump) == 0:
+            print("Coredump not found")
+            return 0
+
+        print("Coredump found")
+        des_path = os.path.join(os.getcwd())
+        copy_cmd = "cp " + coredump[0] + " " + des_path
```

Comment:
why extract array coredump index 1 only?

Reply:
> why extract array coredump index 1 only?

@nowhszh  It's a nice catch, addressed, please review again.